Mathlib/Order/WithBot.lean

```diff
@@ -584,6 +584,15 @@ instance inhabited : Inhabited (WithTop α) :=
 instance nontrivial [Nonempty α] : Nontrivial (WithTop α) :=
   Option.nontrivial
 
+open Function
+
+theorem coe_injective : Injective (fun (a : α) => (a : WithTop α)) :=
```

Comment:
```suggestion
theorem coe_injective : Injective ((↑) : α → WithTop α) :=
```
is a better term to work with in some cases

Reply:
Ok, then I should do the same change to `WithBot`